test/metadata_closure_test.js

```diff
@@ -4,15 +4,15 @@ describe('A test', {
   metadata: { a: 1 },
 
   tests: function() {
-    it('should be able to access its own metadata, given by the suite', function(_metadata) {
-      expect(_metadata).to.eql({ a: 1 });
+    it('should be able to access its own metadata, given by the suite', function() {
+      expect(this.test.metadata).to.eql({ a: 1 });
```

Comment:
I would need to look into this more, but, from the UI layer, I might not be able to change what Context gets sent when the test runs. That might involve modifying the Mocha main file itself or, at the very least, getting a modified version of (this)[https://github.com/mochajs/mocha/blob/958fbb4509eb4e28eee6623b297e60fa48ebeffe/lib/context.js#L22-L35] Context function to call somewhere else